todo/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
If you don't have tests in your project, you are fell free to remove this empty file from project.